### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by mkatenolan** [fac19/week3-JICG#31]
Great that you have labeled most of your inputs but the main input field (where you write your todo) is missing one and it feels quite important! 

I'm assuming that's because you don't want to display it. In that case, an option would be having a label and hiding it visually so at least a screen reader would detect it. Take a look here:
https://www.w3.org/WAI/tutorials/forms/labels/
